lambdaclass/curse_of_myrra
# Spawn Bots visibility bug fix

Closes #899 

The way of handling the backend events together with the instantiation of the SpawnBot instance was resulting in a few scenarios in which only the player that spawned a bot was able to see them in the game.

How to test:

1. Start a new game, with at least two players.
2. Spawn a few bots with one of the players.
3. Verify that all the players in the game can see the bots.
4. Repeat with another player
